homeassistant/components/onkyo/strings.json

```
@@ -0,0 +1,58 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "menu_options": {
+          "manual": "Manual entry",
+          "eiscp_discovery": "Use Onkyo discovery"
```

```suggestion
          "eiscp_discovery": "Onkyo discovery"
```
For consistency with `Manual entry` above. Alternatively we could change the other one to something like `Enter manually`.